moveit_planners/pilz_industrial_motion_planner/CMakeLists.txt

```diff
@@ -1,512 +1,225 @@
-    test/unittest_trajectory_generator_circ.test
-    test/unittest_trajectory_generator_circ.cpp
-  )
-  target_link_libraries(unittest_trajectory_generator_circ
-    ${catkin_LIBRARIES}
-    ${${PROJECT_NAME}_testutils_LIBRARIES}
-    ${PROJECT_NAME}_testhelpers
-  )
-
-  # trajectory generator lin Unit Test
-  add_rostest_gtest(unittest_trajectory_generator_lin
-    test/unittest_trajectory_generator_lin.test
-    test/unittest_trajectory_generator_lin.cpp
-  )
-  target_link_libraries(unittest_trajectory_generator_lin
-    ${catkin_LIBRARIES}
-    ${${PROJECT_NAME}_testutils_LIBRARIES}
-    ${PROJECT_NAME}_testhelpers
-  )
-
-  # trajectory generator ptp Unit Test
-  add_rostest_gtest(unittest_trajectory_generator_ptp
-    test/unittest_trajectory_generator_ptp.test
-    test/unittest_trajectory_generator_ptp.cpp
-  )
-  target_link_libraries(unittest_trajectory_generator_ptp
-    ${catkin_LIBRARIES}
-    ${PROJECT_NAME}_testhelpers
-  )
-
-  # Command Planner Unit Test
-  add_rostest_gtest(unittest_${PROJECT_NAME}
-    test/unittest_${PROJECT_NAME}.test
-    test/unittest_${PROJECT_NAME}.cpp
-  )
-  target_link_libraries(unittest_${PROJECT_NAME}
-    ${catkin_LIBRARIES}
-    ${PROJECT_NAME}_lib
-  )
-
-  # JointLimits Unit Test
-  add_rostest_gtest(unittest_joint_limit
-    test/unittest_joint_limit.test
-    test/unittest_joint_limit.cpp
-  )
-  target_link_libraries(unittest_joint_limit
-    ${catkin_LIBRARIES}
-    ${PROJECT_NAME}_lib
-  )
-
-  # JointLimitsAggregator Unit Test
-  add_rostest_gtest(unittest_joint_limits_aggregator
-    test/unittest_joint_limits_aggregator.test
-    test/unittest_joint_limits_aggregator.cpp
-  )
-  target_link_libraries(unittest_joint_limits_aggregator
-    ${catkin_LIBRARIES}
-    ${PROJECT_NAME}_lib
-  )
-
-  # JointLimitsContainer Unit Test
-  catkin_add_gtest(unittest_joint_limits_container
-    test/unittest_joint_limits_container.cpp
-  )
-  target_link_libraries(unittest_joint_limits_container
-    ${catkin_LIBRARIES}
-    ${PROJECT_NAME}_lib
-  )
-
-  # JointLimitsValidator Unit Test
-  catkin_add_gtest(unittest_joint_limits_validator
-    test/unittest_joint_limits_validator.cpp
-  )
-  target_link_libraries(unittest_joint_limits_validator
-    ${catkin_LIBRARIES}
-    ${PROJECT_NAME}_lib
-  )
+install(
+  DIRECTORY include/
+  DESTINATION include
+)
 
-  # Cartesian Limits Aggregator Unit Test
-  add_rostest_gtest(unittest_cartesian_limits_aggregator
-    test/unittest_cartesian_limits_aggregator.test
-    test/unittest_cartesian_limits_aggregator.cpp
-  )
-  target_link_libraries(unittest_cartesian_limits_aggregator
-    ${catkin_LIBRARIES}
-    ${PROJECT_NAME}_lib
-  )
+ament_export_include_directories(
+  include
+)
+ament_export_libraries(
+  ${PROJECT_NAME}
+  joint_limits_common
+  planning_context_loader_base
+  planning_context_loader_ptp
+  planning_context_loader_lin
+  planning_context_loader_circ
+  sequence_capability
+  trajectory_generation_common
+)
```

Comment:
I don't think these are needed since we export the targets

Reply:
Fixed